travis-ci/terraform-config
# Worker boot and upload scirpt timeouts set for wjb-1 and wjb-2 hosts.

## What is the problem that this PR is trying to fix?

[worker upload script errors on wjb-1, worker boot timeouts on wjb-2](https://ui.honeycomb.io/travis-ci/datasets/worker/result/6dFiDxb9Ly5)

[more here from this week](https://metrics.librato.com/s/metrics/worker.vm.provider.jupiterbrain.boot.timeout?duration=2591640&q=boot.tim&source_filter=%2A)

## What approach did you choose and why?
To keep pods the same, the [worker boot timeout was already added to pod-1](https://github.com/travis-ci/terraform-config/commit/b84fac2423e9c91d0eb2509b4c8b6b2abfc522e4#diff-4cc14e1c0dbd00054b346a0bf0306c7eR17),  adding to pod-2 graph dir. 

## How can you test this?
This has been applied manually in the `travis-worker-production-*` config files on both `wjb-*` hosts and those production workers were restarted.  We will watch both spaces (honeycomb & librato) to see if this helps requeues. 

**Please note that configs have not been changed for any custom workers, (as they were not experiencing timeouts), though this PR would add those changes in terraform**

## What feedback would you like, if any?
Again, the workflow is NOT to make the changes via a `terraform apply` until [graph dir refactoring is done](https://github.com/travis-ci/terraform-config/pull/435) and we can better predict apply behavior on the vCenters (without outages) . 

This PR is only for documentation on whats living on the system only.
